### ISSUE TITLE
Disable device

### ISSUE THREAD

**Issue by ryanfelder** [elementary/switchboard-plug-sound#190]
I have a bunch of audio devices that I never use. It would be nice to be able to easily disable them here so they don't clutter up my device lists.

Perhaps an 'enabled devices' section at the top with 'disable' buttons, a horizontal line, and then 'disabled devices' at the bottom with 'enable' buttons, not entirely unlike bluetooth pairing.

**Comment by walidsaleh** [comment]
yes, it would be nice to toggle between enabled & disabled state for each device.
in deepin linux, in the sound section, there is a subsection for devices where you can toggle each one of them.
